python/dask_cudf/dask_cudf/backends.py

```diff
@@ -256,6 +258,51 @@ def is_categorical_dtype_cudf(obj):
     return cudf.utils.dtypes.is_categorical_dtype(obj)
 
 
+try:
+    from dask.dataframe.dispatch import percentile_dispatch
+
+    @percentile_dispatch.register((cudf.Series, cp.ndarray, cudf.Index))
+    def percentile_cudf(a, q, interpolation="linear"):
+        a = cudf.Series(a)
```

Comment:
What are the inputs `a` and `q`? This looks good, but maybe we can add some clearer documentation on this?

Reply:
Added a comment pointing to the same.